build.gradle

```diff
@@ -46,6 +47,7 @@ apply plugin: 'com.github.hierynomus.license'
 apply plugin: 'com.github.johnrengelman.shadow'
 apply plugin: 'org.lanternpowered.gradle'
 apply plugin: 'org.lanternpowered.run-configs'
+apply plugin: 'de.sebastianboegl.shadow.transformer.log4j'
```

Comment:
@Meronat There is already a transformer being applied that should handle this, it is included in the `org.lanternpowered.gradle` plugin and the transformer is applied in the `shadowJar` task.